sonarr/sonarr.go

```diff
@@ -398,3 +398,19 @@ func (s *Sonarr) GetHistory(maxRecords int) (*History, error) {
 
 	return &history, nil
 }
+
+// Fail marks the given history item as failed by id.
+func (s *Sonarr) Fail(historyID int64) error {
+	if historyID < 1 {
+		return fmt.Errorf("invalid history ID: %d", historyID)
```

Comment:
The `ErrRequestError` variable does not exist in the Sonarr package, so perhaps lets move that variable into `starr/shared.go` and change the ref in Radarr, and add a ref here.
```golang
fmt.Errorf("%w: invalid history ID: %d", starr.ErrRequestError, historyID)
```